### DISCUSSION TITLE
u-center cannot read NTRIP-Caster data

### DISCUSSION THREAD

**Discussion by karlranseierausrom** [semuconsulting/PyGPSClient#105] [Q&A]
I have set up one antenna using u blox gen 9 and PyGPSClient on RPi zero w. I activated NTRIP-Caster in PyGPSClient by setting up the port, username and password. Mode is NTRIP CASTER mode. In that machine I activated UBX and receive good date to that setup.

In another windows based setup I use u-center with another antenna an u blox gen 9. There I used receiver -> NTRIP Client -> enter correct ip, password and so on. Pressing on "update source table" I do receive the right mount point and can retreive data by pressing "mount point details". 

When I press ok to run the windows antenna in rover mode, it gives an error: "unknown header received in continous data reception", this is repeated all the time.

What may I try?

**Comment by karlranseierausrom** [comment]
When disconnecting, activating and deactivating nmea in the caster settings, than reconnecting the caster, than it works.

**Reply by karlranseierausrom** [reply]
I am sorry, but I cannot really reproduce this solution.

Sometimes it works. Everytime I start u-center it tells me that it cannot read some characters. Sometimes it is suddenly working.

Now I found with help the logfiles of my rover, it says the same, I Think:

`process[ntrip_client/ntrip_client_node-3]: started with pid [63603]
[ERROR] [1705181660.305617]: Unable to read response from server at http://192.168.178.34:2101
[ERROR] [1705181660.310365]: Exception: 'utf-8' codec can't decode byte 0xd3 in position 0: invalid continuation byte
[ERROR] [1705181660.315789]: Unable to connect to NTRIP server
[INFO] [1705181660.320071]: Stopping RTCM publisher
[INFO] [1705181660.324161]: Disconnecting NTRIP client [...]
`

**Reply by semuadmin** [reply]
Unfortunately I'm not able to provide support for u-center - you may want to raise a query in the [u-blox support forum](https://portal.u-blox.com/s/), but they'll probably want to see a log of the incoming raw (binary) data.

`0xd3` does represent the first byte in an RTCM message so this at least looks correct.
